packages/orchestrator-ui-components/package.json

```diff
@@ -12,27 +12,23 @@
         "build-storybook": "storybook build"
     },
     "dependencies": {
-        "@elastic/eui": "^92.1.0",
         "@emotion/css": "^11.11.2",
         "@emotion/react": "^11.11.1",
         "@graphql-typed-document-node/core": "3.2.0",
+        "@rtk-query/graphql-request-base-query": "^2.3.1",
         "axios": "^1.6.5",
         "graphql-request": "^6.1.0",
         "invariant": "^2.2.4",
         "moment": "2.29.4",
         "moment-timezone": "^0.5.43",
-        "next": "^14.0.4",
```

Comment:
Since we use the next router, part of next. I think we should keep that one in the package.json